packages/puppeteer-core/src/bidi/Browser.ts

```diff
@@ -111,6 +121,7 @@ export class BidiBrowser extends Browser {
   #browserContexts = new WeakMap<UserContext, BidiBrowserContext>();
   #target = new BidiBrowserTarget(this);
   #cdpConnection?: CdpConnection;
+  #networkEnabled = true;
```

Comment:
We always expect this to be set in the constructor so lets not set a default.
```suggestion
  #networkEnabled: boolean;
```